src/routes/+page.server.ts

```diff
@@ -0,0 +1,19 @@
+import contentful from "contentful";
+import type { ContentfulClientApi } from "contentful";
+import { CONTENTFUL_SPACE_ID, CONTENTFUL_DELIVERY_API_TOKEN } from "$env/static/private";
+
+// TODO: refactor to a reusable helper function that wraps contenfulClient
+export async function load() {
+  const contentfulClient: ContentfulClientApi<undefined> = contentful.createClient({
+    space: CONTENTFUL_SPACE_ID,
+    accessToken: CONTENTFUL_DELIVERY_API_TOKEN,
+  });
+
+  const title = await contentfulClient.getEntry("5k6saMkIV7PHFhNDxmZiIG");
+  const body = await contentfulClient.getEntry("1VCVunA3sIV5Ch1qGHEKZZ");
+
+  return {
+    title: title.fields.body,
+    body: body.fields.body,
```

Comment:
I'm getting TypeScript errors right now supplying these as arguments to `<ContentfulRichText />` and cannot figure out for the life of me what to update to based on the type definitions in Contentful

Reply:
This is the type you're looking for:

```typescript
import type { Document } from "@contentful/rich-text-types";
```

Then you can use `as` for now to assert the type. We'll move this over to fully-typed GraphQL queries eventually anyway (GraphQL queries will let us grab all the data we need for a page in a single request).

Reply:
Actually, we can get rid of the `as`! Try something like this:

```typescript
import contentful, { type ContentfulClientApi, type EntryFieldTypes } from "contentful";

// ...

type EntrySkeleton = {
  contentTypeId: "testRichText",
  fields: { body: EntryFieldTypes.RichText }
};

// ...

export const load = async () => {
  // ...
  const title = await contentfulClient.getEntry<EntrySkeleton>("5k6saMkIV7PHFhNDxmZiIG");
  const body = await contentfulClient.getEntry<EntrySkeleton>("1VCVunA3sIV5Ch1qGHEKZZ");

  return {
    title: title.fields.body,
    body: body.fields.body,
  };
};
```